examples/src/main/java/org/apache/spark/examples/mllib/JavaGaussianMixtureExample.java

```diff
@@ -44,8 +44,9 @@ public static void main(String[] args) {
         public Vector call(String s) {
           String[] sarray = s.trim().split(" ");
           double[] values = new double[sarray.length];
-          for (int i = 0; i < sarray.length; i++)
+          for (int i = 0; i < sarray.length; i++) {
```

Comment:
This is fine, but it wasn't the analysis problem. A lot of the examples don't `stop()` the context at the end, and this was one of about 30.